fix up section about conflict strategies on the client (#640)
aerogear/mobile-docs: modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc

If none is provided, by default, the `clientVersionWins` strategy is used. This means the SDK will automatically override the server data with the current client data. 

> It is very easy to provide a custom conflict resolution strategy. It takes two parameters, `strategies` and `default`.

Reviewer: ```suggestion
To provide a custom conflict resolution strategy, you only need to specify two parameters:
```

Author: I won't take this suggestion, but only because you've highlighted a different issue with what I wrote. Update on the way.

Reviewer: yeah, try to avoid "it's very easy to do anything" - we've got marketing for that :wink:

> * strategies - a dictionary object defining the custom behaviour for any mutation you which to specify

Reviewer: ```suggestion
* strategies - a dictionary object defining the custom behavior for any mutation you want to specify
```
not sure about this one

Author: `behavior` are we using US spelling in general?

Reviewer: yes :smiley_cat:
